### PR TITLE
Add "git_pr" in payload.

### PR DESCRIPTION
**Pull request by GeetikaBatra**
This PR adds "git_pr" in the payload for version
notifications. It represents the pull request opened
for the subsequent change.

Signed Off by: Geetika Batra<gbatra@redhat.com>

### CODE REVIEW

**Comment by GeetikaBatra** [comment]
[test]

**Comment by nurali-techie** [comment]
[test]

**Comment by GeetikaBatra** [comment]
[test]

File: template/analytics.notify.version/body.html [hunk 1]
```diff
@@ -142,6 +142,20 @@
...
+    text-align:center;
+    color: #363636;">
+<span>
+    <span style="font-weight: 600;">
+        Pull Request:
+      </span>
+      <a href={{.custom.git_pr}} style="color: #0088ce; text-decoration: none;">{{.custom.git_pr}}</a>
+</span>
+</div>
+    {{end}}
```
Comment (nurali-techie) [inline template/analytics.notify.version/body.html]: @GeetikaBatra have you review this template htlm file from your team?

**Comment by GeetikaBatra** [comment]
@nurali-techie Sure On it.

**Comment by nurali-techie** [comment]
Issue created to address failing **ci.centos.org PR build (coverage)** check - https://github.com/fabric8-services/fabric8-notification/issues/100

**Comment by GeetikaBatra** [comment]
:+1:

**Comment by nurali-techie** [comment]
> @nurali-techie Sure On it.

@GeetikaBatra I got confused, you are sure on changes ??  Or you will get review done with your teammates ??

**Comment by GeetikaBatra** [comment]
@nurali-techie. Will get review from team. I have pinged Yusuf on channel. I cannot add him as a reviewer here since he is not part of this org.

**Comment by yzainee** [comment]
LGTM @nurali-techie @geetikabatra

**Pull request merged**